integration/embedded_derp_test.go

```diff
@@ -185,6 +188,34 @@ func derpServerScenario(
 
 	t.Logf("Run2: %d successful pings out of %d", success, len(allClients)*len(allHostnames))
 
+	hsServer, err := scenario.Headscale()
+	assertNoErrGetHeadscale(t, err)
+
+	derpRegion := tailcfg.DERPRegion{
+		RegionCode: "test-derpverify",
+		RegionName: "TestDerpVerify",
+		Nodes: []*tailcfg.DERPNode{
+			{
+				Name:             "TestDerpVerify",
+				RegionID:         900,
+				HostName:         hsServer.GetHostname(),
+				STUNPort:         3478,
+				STUNOnly:         false,
+				DERPPort:         443,
+				InsecureForTests: true,
+			},
+		},
+	}
+
+	fakeKey := key.NewNode()
+	DERPVerify(t, fakeKey, derpRegion, false)
+
+	for _, client := range allClients {
+		nodeKey, err := client.GetNodePrivateKey()
+		assertNoErr(t, err)
+		DERPVerify(t, *nodeKey, derpRegion, true)
+	}
```

Comment:
I do not think this is necessary? if the verify flag is on, wouldnt all the tests just fail later if they are not able to connect to the DERPs since it is now forcing verification?